ui/narrative/methods/run_maxbin2/spec.json

```diff
@@ -42,7 +42,7 @@
             "default_values" : [ "" ],
             "field_type" : "text",
             "text_options" : {
-              "valid_ws_types" : ["KBaseFile.PairedEndLibrary", "KBaseFile.SingleEndLibrary"]
+              "valid_ws_types" : ["KBaseFile.PairedEndLibrary", "KBaseFile.SingleEndLibrary", "KBaseAssembly.PairedEndLibrary", "KBaseAssembly.SingleEndLibrary"]
```

Comment:
needs tests for KBaseAssembly.PairedEndLibrary and SingleEndLibrary objects

Reply:
Can we still upload KBaseAssembly types?

Reply:
I don't know. But if you want to include new types, you should add tests.

Reply:
I agree.  I can't use the read uploader though.  I'll have to go through the pain-in-the-neck process of creating KBaseAssembly type read libraries myself in the unit tests.  Hold on...